packages/diagnostic/src/disgnostic.ts

```diff
@@ -0,0 +1,19 @@
+import Debugger from 'debug';
+
+const debug = Debugger('rehearsal:diagnostic');
+
+export async function diagnose(srcDir: string): Promise<string> {
```

Comment:
the filename has a typo and should just be renamed the same as the export. `diagnose.ts`